snforge_std/src/trace.cairo

```diff
@@ -0,0 +1,60 @@
+use starknet::testing::cheatcode;
+use starknet::ContractAddress;
+use super::PrintTrait;
+
+#[derive(Drop, Serde, PartialEq)]
+struct CallEntryPoint {
+    entry_point_type: EntryPointType,
+    entry_point_selector: felt252,
+    calldata: Array<felt252>,
+    storage_address: ContractAddress,
+    caller_address: ContractAddress,
+    call_type: CallType,
+}
```

Comment:
Basically copied struct from blockifier, but without `initial_gas` as it has weird values for now - #1418